test/conformance/tests/sriov_operator.go

```diff
@@ -1968,3 +1977,24 @@ func setSriovOperatorSpecFlag(flagName string, flagValue bool) {
 
 	}
 }
+
+func getNodeDrain() (bool, error) {
+	sriovOperatorConfg := &sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: "default", Namespace: "openshift-sriov-network-operator"}, sriovOperatorConfg)
+	return sriovOperatorConfg.Spec.DisableDrain, err
+}
+
+func setNodeDrain() error {
+	sriovOperatorConfg := &sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: "default", Namespace: "openshift-sriov-network-operator"}, sriovOperatorConfg)
+	if err != nil {
+		return err
+	}
+	sriovOperatorConfg.Spec.DisableDrain = true
+	err = clients.Update(context.TODO(), sriovOperatorConfg)
+	if err != nil {
+		return err
+	}
+	err = clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: "default", Namespace: "openshift-sriov-network-operator"}, sriovOperatorConfg)
```

Comment:
there is a variable for the namespace please use it

Reply:
Done